src/semver.rs

```diff
@@ -53,6 +54,13 @@ impl Version {
             }),
         }
     }
+
+    /// Is the current version's major component 0? If so, rules are applied differently in `bump`.
```

Comment:
What are thoughts on moving this comment to `bump` since it seems to be describing the behavior of `bump` and not `is_0`?  It's possible we might use this in more than `bump` in the future.

Similarly, if `bump` is updated in the future (name change, does not use `is_0`, removed, etc.), the comment will no longer be valid.

Reply:
(I won't hold the PR up over this comment, but I do think it's worth considering)

Reply:
I added more detail to the documentation both to address your concern and answer Ben's question in code.